packages/client/modules/email/components/SummaryEmail/ExportToCSV.tsx

```diff
@@ -190,7 +190,9 @@ class ExportToCSV extends Component<Props> {
     const {endedAt, team, meetingType} = newMeeting
     const {name: teamName} = team
     const label = meetingType[0].toUpperCase() + meetingType.slice(1)
-    const parser = new Parser()
+    const withBOM = true
+    const opts = {withBOM}
+    const parser = new Parser(opts)
```

Comment:
Can you please compact this to 1 line, not 3?

Reply:
Is it okay now?

Reply:
Can you make this...

```javascript
const parser = new Parser({ withBOM: true})
```
 
...instead?